aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -154,7 +154,13 @@ Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& ma
   at::ScalarType scalar_type = self.scalar_type();
   c10::MaybeOwned<Tensor> self_;
   if (&result != &self) {
-#if defined(CUDA_VERSION) && CUDA_VERSION >= 11040 && !defined(_MSC_VER)
+    const char* env_value = std::getenv("DISABLE_ADDMM_CUDA_LT");
```

Comment:
please make it a static var so that `getenv` is called once and not every time on a hot path (you can look at other places how env vars are checked)